gnomad/resources/grch38/gnomad.py

```diff
@@ -15,9 +15,8 @@
 )
 from gnomad.sample_qc.ancestry import POP_NAMES
 from gnomad.utils.annotations import (
-    add_gks_va,
-    add_gks_vrs,
-    get_gks,
+    add_gks_va_py,
+    add_gks_vrs_py,
     gks_compute_seqloc_digest,
```

Comment:
the function from gks_compute_seqloc_digest got coded into add_gks_vrs_py , so we definitely can remove this

currently only calls vrs_py and va_py from this

one question though - while working on them I added a _py suffix to differentiate them, since they're all in Python and not Hail anymore. Should I keep the suffix or is it unnecessary